src/k8s-engine/wsl.ts

```diff
@@ -1131,6 +1162,9 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     try {
       this.setState(K8s.State.STOPPING);
       await this.progressTracker.action('Stopping Kubernetes', 10, async() => {
+        if (this.#currentContainerEngine !== ContainerEngine.MOBY) {
+          await this.execCommand('/usr/local/bin/wsl-service', '--ifstarted', 'buildkitd', 'stop');
```

Comment:
We should always stop services, even if we don't expect them to have started (to ensure the real state is closer to our expected state).

Also, this needs to be in the `if(isDistroRegistered)` clause, to ensure we don't accidentally _start_ the distro if it's already dead.

Reply:
Makes sense.